extensions/dacpac/src/test/index.ts

```diff
@@ -14,9 +14,9 @@ const testOptions: any = {
 	timeout: 60000
 };
 
-const coverageConfig: any = {
-	coverConfig: '../../coverageConfig.json'
-};
+// const coverageConfig: any = {
+// 	coverConfig: '../../coverageConfig.json'
+// };
```

Comment:
Why removing?

Reply:
`vscode/lib/testrunner` does not support coverage reports.

Reply:
We decided that because of the test runner errors we rather get everything working then do test coverage.

Reply:
Just delete this then until we figure out what we want to do here IMO